packages/client/tests/functional/metrics/enabled/tests.ts

```diff
@@ -269,9 +269,11 @@ testMatrix.setupTestSuite(
             {
               key: 'prisma_pool_connections_idle',
               labels: {},
-              // This can sometimes be reported as 1 or 0,
+              // This can sometimes be reported as 0, 1 or 2,
               // as metrics are reported asynchronously.
-              value: expect.toBeOneOf([0, 1]),
+              // Note: We want to investigate why these different values are reported in our test setup
+              // https://github.com/prisma/team-orm/issues/587
+              value: expect.toBeOneOf([0, 1, 2]),
```

Comment:
Do you think we could expect any number here, or is it fully limited to this set?

Reply:
That's something to investigate, my guess is that this should cover most cases, because that would be weird if not but possible.